### DISCUSSION TITLE
Any integration/plugin for wordpress

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#8553] [Unsupported Versions]
Hello, is there any integration with wordpress possible with security onion? Trying to gain some more visibility with one of our sites.

**Comment by isaacgolding** [comment]
Can you explain the question further please?

**Reply by thegreekman** [reply]
Like how there is integration with amazon/cisco/etc. Is there anything for wordpress? I would like to be able to see analytics for a website via security onion

**Comment by dougburks** [comment]
I don't see any Filebeat modules for Wordpress:
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-modules.html

Years ago, there was a Wordpress plugin that would send events to syslog. Here's an old blog post from 2010 where I documented how to use it with Security Onion at the time:
https://blog.securityonion.net/2010/02/defense-in-depth-using-ossec-and-other.html

I'm not sure if that plugin is still maintained, but it might give you some ideas.
